docs/user_guide.md

```diff
@@ -1,15 +1,17 @@
 ## Quick Start
-1. Check [dependencies](#dependencies)
-2. Fill the CSV [config file](#csv-config-file)
-3. Generate the [dashboard](#how-to-generate-dashboard)
-4. Profit!
+1. Ensure that you have the necessary [dependencies](#dependencies).
+2. Read up on [How to Generate Dashboard](#how-to-generate-dashboard).
+3. Fill up the CSV [configuration file](#csv-config-file).
+4. Perform the execution to generate the [dashboard](#dashboard).
```

Comment:
Suggestion: This sounds a bit unclear and similar to the 2nd point. Something clearer could be helpful if possible.